### PR TITLE
COMMERCE-9998 On the Global Search, it's not possible to search for existing orders entry using an email address

### PR DESCRIPTION
**Pull request by baharturk**
https://issues.liferay.com/browse/COMMERCE-9998

Sent from #3194

@ MahmoudAzzam

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSearch.testcase [hunk 1]
```diff
@@ -65,6 +65,116 @@ definition {
...
+				newAddressAddress = "Test Address",
+				newAddressCity = "Test City",
+				newAddressCountry = "United States",
+				newAddressName = "Address Name",
+				newAddressZipCode = "Test Zip Code",
+				productName = "U-Joint",
+				productQuantity = "1",
+				productUnitPrice = "24",
+				shippingMethod = "Standard");
+		}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSearch.testcase:114]: @baharturk this can be done using API calls

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSearch.testcase (hunk 1 above)
Comment (baharturk) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSearch.testcase:114]: The aim of this test is to do use global search by passing the email address of the buyer user, but the macro used for placing an order with an API call does not support doing it that way so I'm keeping the test as it is now, we also discussed it with the reviewer.

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**